testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/mdb/timerservice/TimedObjectTimerServiceMDB.java

```diff
@@ -37,7 +37,7 @@ public void createTimer() {
 
     public static boolean awaitTimerCall() {
         try {
-            latch.await(2, TimeUnit.SECONDS);
+            latch.await(5, TimeUnit.SECONDS;
```

Comment:
This doesn't compile, BTW... :)
That said, rather than bumping the wait time, I would suggest using org.jboss.as.test.shared.TimeoutUtil.adjust(...) to scale the timeout value to accommodate laggy test hosts.